pkgs/top-level/all-packages.nix

```diff
@@ -33845,9 +33845,7 @@ with pkgs;
     xwaylandSupport = false;
   };
 
-  chatterino2 = callPackage ../applications/networking/instant-messengers/chatterino2 {
-    stdenv = if stdenv.hostPlatform.isDarwin then darwin.apple_sdk_11_0.stdenv else stdenv;
-  };
+  inherit (callPackages ../applications/networking/instant-messengers/chatterino2 { } ) chatterino2 chatterino2-unstable chatterino7 chatterino7-unstable;
```

Comment:
Why do we track unstable versions?

Reply:
Hi, thanks for the input!

It is quite normal for people to choose the ["nightly builds"](https://github.com/Chatterino/chatterino2#nightly-build) if they experience any issues as it takes a while between versioned releases. Last release was at [April 24](https://github.com/Chatterino/chatterino2/releases/tag/v2.5.1). 
I followed the [nixpkgs versioning guidelines](https://github.com/NixOS/nixpkgs/tree/master/pkgs#versioning)